components/Pamphlet/Pamphlet.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  Button,
+  Container,
+  Header,
+  Image,
+  List
+ } from 'semantic-ui-react';
+
+import styles from './Pamphlet.less';
+
+export const Pamphlet = (props) => {
+  const { header, price, description } = props;
+
+  return (
+    <div>
```

Comment:
Could this be a fragment? Or is that unnecessary because this component is not being rendered multiple times?